views/author.handlebars

```diff
@@ -5,52 +5,65 @@
   <title>Autor</title>
   <link rel="stylesheet" href="/css/author.css">
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fjalla+One">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.0/css/bulma.min.css">
+
 </head>
 <body>
-  <section class="author-section">
-    <header class="author-header">
-      <div class="logo">
-        <h1><a href="/" class="link-logo">&#60;/CONHECIMENTO LIVRE&#62;</a></h1>
+  <nav class="navbar is-primary" role="navigation" aria-label="main navigation" style=" z-index: 3;">
+    <div class="navbar-brand">
+      <a class="navbar-item link-logo" href="/">
+        <h1>&#60;/CONHECIMENTO LIVRE&#62;</h1>
+      </a>
+    </div>
+  </nav>
+
+  <div class="columns">
```

Comment:
Poderiamos usar outro formato pra estruturar a pagina, da uma olha sobre is-parent e is-child no bulma! 
https://bulma.io/documentation/layout/tiles/

Reply:
Na branch refatora-pagina-conteudo, foi feito com essa estrutura tambem